### ISSUE TITLE
When installing Linkerd from master, we get an error installing linkerd-tap RBAC

### ISSUE THREAD

**Issue by dadjeibaah** [linkerd/linkerd2#2844]
When building and installing Linkerd from current master, we get an error when installing the control plane because of a misconfiguration in `tap-rbac.yaml`
```bash
$ bin/docker-build
$ bin/linkerd install | kubectl apply -f -
$ ....
$The ClusterRoleBinding "linkerd-linkerd-tap" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"ClusterRole", Name:"linkerd-linkerd-controller"}: cannot change roleRef
```

**Comment by ihcsim** [comment]
I'm unable to reproduce this on `master`; i.e. I don't think `master` is broken. Is this caused by an existing `linkerd-linkerd-tap` binding that has a different `roleRef` value on your cluster? Nonetheless, the fix in #2845 looks right to me.

**Comment by dadjeibaah** [comment]
Ahh I think you are right, I think I had a cluster that initially had a `roleRef` that had this fix but wasn't pushed to master of `linkerd-linkerd-controller` then when I re-built the linkerd and tried to install I ran into this issue.
